docs/source/_templates/sphinx_conf.template.py

```diff
@@ -34,12 +34,7 @@
 extensions = [
     "sphinx_design",
     "sphinx.ext.autodoc",
-    "sphinx.ext.mathjax",
-    "sphinx.ext.napoleon",
```

Comment:
This is necessary for numpy docstring rendering with apidoc: https://www.sphinx-doc.org/en/master/usage/extensions/napoleon.html